### DISCUSSION TITLE
Filecoin中文社区正式启动啦 !!!

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community-china#1] [社区公告]
## 👋 欢迎来到Filecoin中文社区!

Community-China 致力于整合 Filecoin 中文社群的挖矿教程、开源项目、工具资源、答疑解惑等等，为中文矿工提供一个相互帮助、共赢发展的平台。培养中文社区的开发者，帮助 Filecoin 生态的创业者，推动 Filecoin 生态和 Web3.0 的发展。

**我们希望矿工和开发者在这里：**
  * 帮助新入门矿工快速熟悉 Filecoin 挖矿硬件配置、集群搭建、运维管理等；
  * 讨论性价比最高的硬件配置，研究如何打造最高效率的挖矿软件，也欢迎社区大神踊跃开源代码；
  * 探讨如何构建安全稳定的存储系统，如何搭建高可用、可扩展的网络架构；
  * 开发者一起探讨如何基于 Filecoin 和 IPFS 构建令人惊叹的 Web3.0 应用；
  * 还有开发、运维、硬件人才招聘版块。
  
**我们希望您在发帖讨论时，遵守以下规则：**
  * 抵制标题党，没有任何实质内容的帖子将被视为水贴，管理员将直接删除，不进行任何通知。
  * 提问时请贴上详细的日志(建议文本方式，便于复制)、系统环境、运行环境、软件版本，问题越详尽，得到解答的可能性越大，提问前先问问自己是否定义清楚了问题；
  * 使用清晰优雅的Markdown排版格式，参照Github的[Markdown基本撰写和格式语法](https://docs.github.com/cn/github/writing-on-github/basic-writing-and-formatting-syntax)；
  * 招聘信息附工作地点和薪资福利；
  * 不讨论币价和炒币。

**Comment by 0xhhhhh** [comment]
恭喜恭喜

**Reply by hredniuniu** [reply]
人人都可以只需要可以需要租赁
手机的微信：x15307438912

**Reply by a191912356** [reply]
WeChat : xiao1234xiao8

**Reply by KaixuanZhao** [reply]
IPFS主网最大雅典娜矿池，RRmine global ，一站式采购托管运营，全部数据链上可查，全网算力领先1400PB，需要参与Datacap真实数据存储的朋友可以在小狐狸浏览器中复制打开https://www.rrmglobal.io/#/invite?invite_code=O3DKS2S0 欢迎加入社群 Wechet：kkmmjjnn00

**Reply by allanQT** [reply]
FIL中国三峡大坝数据中心，顶级数据中心，自建自存业务，享受三峡大坝绿色能源和顶级安保，真实数据存储，顶级合规建设。VX：13177077730

**Comment by billconan** [comment]
能简化到人人挖就好了，参与度才够

**Reply by 0xhhhhh** [reply]
现在要求还是挺高的，起步都要AMD的U

**Reply by KaixuanZhao** [reply]
挖fil可以找我，有优惠

**Reply by KaixuanZhao** [reply]
IPFS主网最大雅典娜矿池，RRmine global ，一站式采购托管运营，全部数据链上可查，全网算力领先1400PB，需要参与Datacap真实数据存储的朋友可以在小狐狸浏览器中复制打开https://www.rrmglobal.io/#/invite?invite_code=O3DKS2S0 欢迎加入社群 Wechet：kkmmjjnn00

**Reply by hredniuniu** [reply]
北美矿业拥有遍布全球19个大型矿场，可同时运行80万台矿机，是全球算力第一矿池，手机一站式云算力挖矿,专注BTC'ETH'FIL等主流货币挖矿服务团队为你解决各种疑惑和问题。
矿机租赁时长6年（2190天）
产出效益高出同行30%-50%
每天收益,随时提现
满算力,无需质押币
专注云算力挖矿,无需购买矿机就能轻松挖矿。
北美矿业官网：www.napmine.com

北美矿业NAP挖矿app下载链接 ： http://nap.sleon.cn/index.html#/register?inviteCode=ehymoc

北美矿业2019年成立于美国波士顿, 已获得美国、澳大利亚等多国合规牌照,实体办事处位于美国马萨诸州波士顿肯德尔广场, 绝无跑路风险。其母公司⸺北美智科国际集团（NAZK),由美国、加拿大顶级投资机构⸺CIBC、Scotia Bank、National Bank、黑石集团联合创办，主要投产于生命科技、人工智能、航天探测和区块链领域，北美智科国际集团是经美国商务部认证的高新技术企业, 旗下多条产品线获得美国FDA检验认证、欧盟CE认证, 安全可靠、实力过硬！

**Reply by hredniuniu** [reply]
官方统一推荐码是ehymoc

**Reply by ii55q** [reply]
这还得看FIDEX 挖矿无门槛 TG群： https://t.me/OnlyFidex_com

**Comment by gong1414** [comment]
希望能把密封流程那个文档完善，扒着源码看伤不起

**Comment by KaixuanZhao** [comment]
挖fil可以找我，VX18686935540

**Comment by ShayneDong** [comment]
哪位大佬给点测试环境的测试币，感谢，地址t3qt42quuxuavup2leks545hos773yjt24xpwdnqkuz6ytythef7t3qhnjbznej5z7sd3arx2ulcamjlrx53ca

**Reply by yunfens** [reply]
项目整体转让，位于广州，fil自建机房，配置，3台空调，50千瓦柴油发电机及配套ups；存储配置：6台36盘14T存储机5台8盘存储机合计3P存储空间，5台3090显卡adm7402，1t，内存，2*8T缓存封装机，每天10T的封装能力，3P存储已经封装了430T，一台节点机 3090显卡，分币系统app及源码，项目整体转让，并免费提供技术支持！机房租金3000每月，可放置6台42U机柜，并可以扩充更多！vx:186-2004-4433

**Comment by ljh8mg1018** [comment]
![2](https://user-images.githubusercontent.com/84759516/129130916-ee2c53c6-203a-4575-aec3-2893398f0045.jpg)
![1628733458189](https://user-images.githubusercontent.com/84759516/129130922-78fc426e-0f7f-46d0-8af2-abba7e89ed92.jpg)
有前輩可以解答嗎!!我不知道發生甚麼!!沒再封裝

**Comment by KaixuanZhao** [comment]
RRMine，雅典娜矿池，全网算力领先1000PB，需要B端C端方案的滴滴我，交流学习，欢迎加入社群Wechet：kkmmjjnn00

**Comment by KaixuanZhao** [comment]
RRMine，雅典娜矿池，全网算力领先1000PB，需要B端C端方案的滴滴我，交流学习，欢迎加入社群Wechet：kkmmjjnn00

**Comment by jiangopen8** [comment]
https://www.youtube.com/watch?v=tzxjK9PLHbk

**Comment by jiangopen8** [comment]
Dicomnetwork is a storage and diagnostic network base on filecoin for patients and doctors worldwide

**Comment by GZpowerleader** [comment]
有没有需要硬件的 VX18589289061

**Comment by SeanLau0811** [comment]
file联合挖矿，云算力销售，节点可查，香港IDC机房，安全有保障。微信滴滴：xiaofu0811

**Comment by lee620422** [comment]
filecoin联合挖矿，算力销售，节点可查，香港IDC机房，安全有保障。PM：微信HBT_CNN

**Comment by zhaobo443** [comment]
合肥矿场，联通IDC机房，业务范围：设备制造，矿池搭建，数据中心，软件运维，算力销售，联合挖矿。靠谱。V:zhaobo9308

**Comment by zhaobo443** [comment]
合肥矿场，联通IDC机房，业务范围：设备制造，矿池搭建，数据中心，软件运维，算力销售，联合挖矿。靠谱。V:zhaobo9308

**Comment by JairoTan** [comment]
能不能把这些广告的全kill了

**Comment by Awe1-s** [comment]
filecoin 联合挖矿 算力封装 矿池搭建 算力恢复 节点可查 软件运维 深圳IDC机房 熊市囤币 你懂的V:Believeit_Fil

**Comment by RichByDMC** [comment]
挖FIL的伙伴们看过来，一个火爆全网的全新存储项目自带公链钱包NFT等完整生态DMC，唯一能够超过Fil的项目诞生了。主要还能双挖FIL和DMC~~有想法的可以V我：dmctorich
